src/dash-adapter.js

```diff
@@ -784,6 +796,37 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
     }, (segmentDuration + 1) * 1000);
   }
 
+  _switchFromDynamicToStatic(): void {
+    DashAdapter._logger.debug('Switch from dynamic manifest to static');
+    const videoFullDuration = this._getLiveEdge() - this._seekRangeStart;
```

Comment:
you can simplify to 
this.videoElement.currentTime - this._seekRangeStart

Reply:
just note that you need to save this into a variable BEFORE load is called, otherwise this._seekRangeStart will be reset